### PR TITLE
Jamdagni86 fix for #17 (Support tuple args in lambda)

### PR DESCRIPTION
**Pull request by alanjds**
Rebased @Jamdagni86 work on https://github.com/google/grumpy/pull/265/ over #87 

Kudos  @Jamdagni86 for the code and @trotterdylan for original comments.

### CODE REVIEW

File: grumpy-tools-src/grumpy_tools/compiler/block.py [hunk 1]
```diff
@@ -364,16 +370,40 @@ def __init__(self, node):
...
     node_args = node.args
-    args = [a.arg for a in node_args.args]
+    args = []
+    for arg in node_args.args:
+      if isinstance(arg, ast.Tuple):
+        args.append(arg.elts)
+      else:
+        args.append(arg.arg)
+     # args = [a.arg for a in node_args.args]
+
```
Comment (RezaGhanbari) [inline grumpy-tools-src/grumpy_tools/compiler/block.py]: args = [arg.elts if isinstance(arg, ast.Tuple) arg.arg for arg in node_args.args]

Comment (alanjds) [reply grumpy-tools-src/grumpy_tools/compiler/block.py]:
```python
args = [(arg.elts if isinstance(arg, ast.Tuple) else arg.arg) for arg in node_args.args]
```

I am wondering if this version is more readable than the original

Comment (RezaGhanbari) [reply grumpy-tools-src/grumpy_tools/compiler/block.py]: It's just the pythonic way, but you'r right . the original is more readable.

**Comment by alanjds** [comment]
Thanks @jamdagni86.

**Comment by jamdagni86** [comment]
you're welcome @alanjds 👍

**Pull request merged**